app/src/main/java/io/gnosis/safe/Tracker.kt

```diff
@@ -78,8 +78,10 @@ class Tracker private constructor(context: Context) {
         }
     }
 
-    fun logException(exception: Throwable) {
-        FirebaseCrashlytics.getInstance().recordException(exception)
+    fun logException(exception: Throwable, trackIt : Boolean = true) {
```

Comment:
I think this change should be reverted. If exception should not be trackable it is better not to call logException at all. logException(e, false) is missleading.

Reply:
Done